<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestEditor</class>
 <widget class="QWidget" name="RequestEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="requestNameEdit">
        <property name="placeholderText">
         <string>Request Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QComboBox" name="methodCombo">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string>GET</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="urlEdit">
           <property name="placeholderText">
            <string>Url</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendButton">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="widget_5" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="switchRequestButton">
           <property name="text">
            <string>Request</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">requestResponseButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="switchResponseButton">
           <property name="text">
            <string>Response</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">requestResponseButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="loadingSpinnerStack">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="loadingSpinnerContainer"/>
     <widget class="QWidget" name="requestResponseStackContainer">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="requestResponseStack">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="requestContainerWrapper"/>
         <widget class="QWidget" name="responseContainerWrapper">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QStackedWidget" name="responseLoaderStack">
             <widget class="QWidget" name="responseLoaderContainer">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                <widget class="QWidget" name="widget_2" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item alignment="Qt::AlignHCenter">
                   <widget class="QWidget" name="responseLoaderWrap" native="true">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>100</width>
                      <height>100</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>100</height>
                     </size>
                    </property>
                    <property name="baseSize">
                     <size>
                      <width>100</width>
                      <height>100</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="cancelRequestButton">
                    <property name="text">
                     <string>Cancel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="responseContainerInnerWrap"/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="requestResponseButtonGroup"/>
 </buttongroups>
</ui>
